src/components/ObsDetails/ObsDetails.js

```diff
@@ -334,28 +334,11 @@ const ObsDetails = ( ): Node => {
         {displayPhoto()}
         <View className="flex-row my-5 justify-between mx-3">
           {showTaxon()}
-          <View>
-            <ActivityCount
-              color={colors.darkGray}
-              count={observation.identifications.length}
-              icon="identification-solid"
-              accessibilityLabel={
-                t( "x-identifications", { count: observation.identifications.length } )
-              }
-            />
-            <ActivityCount
-              color={colors.darkGray}
-              count={observation.comments.length}
-              icon="comments-filled-in"
-              accessibilityLabel={
-                t( "x-comments", { count: observation.comments.length } )
-              }
-            />
-            <QualityGradeStatus
-              qualityGrade={checkCamelAndSnakeCase( observation, "qualityGrade" )}
-              color={colors.darkGray}
-            />
-          </View>
+          <ObsStatus
+            layout="vertical"
+            observation={observation}
+            color={colors.darkGray}
```

Comment:
I would rather make this the default color inside ObsStatus (and all its children) and override only with a "white" boolean. So, that there is no possibility for now to use another color.